token/js/src/extensions/tokenMetadata/actions.ts

```diff
@@ -0,0 +1,133 @@
+import type { ConfirmOptions, Connection, PublicKey, Signer, TransactionSignature } from '@solana/web3.js';
+import type { Field } from '@solana/spl-token-metadata';
+import {
+    createInitializeInstruction,
+    createEmitInstruction,
+    createUpdateFieldInstruction,
+} from '@solana/spl-token-metadata';
+import { sendAndConfirmTransaction, Transaction } from '@solana/web3.js';
+
+import { TOKEN_2022_PROGRAM_ID } from '../../constants.js';
+import { getSigners } from '../../actions/internal.js';
+
+/**
+ * Initializes a TLV entry with the basic token-metadata fields.
+ *
+ * @param connection       Connection to use
+ * @param payer            Payer of the transaction fees
+ * @param updateAuthority  Update Authority
+ * @param mint             Mint Account
+ * @param mintAuthority    Mint Authority
+ * @param name             Longer name of token
+ * @param symbol           Shortened symbol of token
+ * @param uri              URI pointing to more metadata (image, video, etc)
+ * @param multiSigners     Signing accounts if `authority` is a multisig
+ * @param confirmOptions   Options for confirming the transaction
+ * @param programId        SPL Token program account
+ *
+ * @return Signature of the confirmed transaction
+ */
+export async function initializeTokenMetadata(
```

Comment:
nit: I would rather you name these according to the same pattern we have in the Rust client, ie:

- `tokenMetadataInitialize(..)`
- `tokenMetadataUpdateField(..)`
- ...